docs/sources/clients/promtail/scraping.md

```diff
@@ -422,7 +422,7 @@ scrape_configs:
 ```
 
 Only `api_token` and `zone_id` are required.
-Refer to the [Cloudfare](../configuration/#cloudflare) configuration section for details.
+Refer to the [Cloudfare](https://grafana.com/docs/loki/latest/clients/promtail/configuration/#cloudflare) configuration section for details.
```

Comment:
ditto

Reply:
Yeah using Hugo relrefs (https://gohugo.io/functions/relref/) would be the best solution. The links will be checked when you run `make docs`

Reply:
Additionally if we add https://github.com/grafana/loki/pull/6569 to CI, you can get anchor checking too

Reply:
Thanks.